dmoj/checkers/linecount.py

```diff
@@ -1,19 +1,40 @@
+from __future__ import print_function
+import sys
+
 from dmoj.result import CheckerResult
 
+equal = lambda p, j: p.strip() == j.strip()
 verdict = u"\u2717\u2713"
 
 def check(process_output, judge_output, point_value, feedback=False, **kwargs):
+
     process_lines = filter(None, process_output.strip().split("\n"))
     judge_lines = filter(None, judge_output.strip().split("\n"))
 
+    # Extraneous output
+    if len(process_lines) > len(judge_lines):
+        return False
+
+    match = equal
+
+    # Overload lambda - passed as a string literal
+    if 'match' in kwargs:
+        try:
+            # Skips assignment if eval fails
+            match = eval(kwargs['match'])
+        except Exception as e:
+            print(e.__doc__, e.message, file=sys.stderr, sep='\n')
```

Comment:
In any case, don't catch the exception and just let it propagate into internal error, which this is, instead of silently eating it.